install.sh

```diff
@@ -177,6 +177,23 @@ nvm_check_global_modules() {
   fi
 }
 
+nvm_bash_completion() {
+  if [ -f "$HOME/.bashrc" ] || [ -f "$HOME/.bash_profile" ] || [ -f "$HOME/.zshrc" ] ; then
```

Comment:
That's true - in your current code, you're doing up to 6 checks - 3 files twice, plus 1 `bash_completion`. With what I'm proposing, you'd do up to 6 as well - 3 files, plus 3 `bash_completion`. So I suppose it'd work out to be the same. However, you could use a local variable and set it to `true` when the file downloads, and then you'd only ever do up to 4 checks.